### PR TITLE
[TCFMM-24] Add sanity checks

### PR DESCRIPTION
**Pull request by dcastro**
## Description

Adds a couple of sanity checks.

I couldn't think of anything else to add, since we encoded most invariants in the `Invariants` module anyway.

## Related issue(s)

Resolves TCFMM-24

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

[//]: # (Add more docs here if you have them in the repository)
- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [Specification](../tree/master/docs/specification.md)
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

[//]: # (Update link to style guide if necesary or remove if it's not present)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

**Comment by dcastro** [comment]
@Martoon-00 can i get a quick review here please? ^^

File: ligo/helpers.mligo [hunk 1]
```diff
@@ -37,9 +37,19 @@
...
-        (Bitwise.shift_left (liquidity * sqrt_price_old.x80) 80n)
-        ((Bitwise.shift_left liquidity 80n) + dx * sqrt_price_old.x80)}
+    let sqrt_price_new =
+        {x80 = floordiv
+            (Bitwise.shift_left (liquidity * sqrt_price_old.x80) 80n)
+            ((Bitwise.shift_left liquidity 80n) + dx * sqrt_price_old.x80)
+        } in
+#if DEBUG
+    let _ : unit =
+        if sqrt_price_new <= sqrt_price_old
```
Comment (rinn7e) [inline ligo/helpers.mligo:47]: @dcastro  Is there a case where the new price stay the same?

**Review by rinn7e — APPROVED** [review]
Looks good to me, only have one question.

File: ligo/helpers.mligo (hunk 1 above)
Comment (dcastro) [reply ligo/helpers.mligo:47]: Yeah, technically you _can_ swap 0 tokens (tho I'm not sure why anyone would do this lol)

**Pull request merged**